client/components/product-card/style.scss

```diff
@@ -204,10 +204,16 @@
 	line-height: 20px;
 	color: var( --color-text-subtle );
 
-	a {
+	a:not( .button ) {
 		@extend %product-card-link-plain;
 	}
-
+	.button {
+		margin-left: auto;
```

Comment:
I'd recommend against the block centering strategy here. Can we instead center it as an inline element? So you only have to put `text-align: center` on the wrapper element of the button.

Reply:
No since the wrapper element is also responsible for the styling of the description text.

Reply:
Let's add another wrapper that wraps only the button then. In that case we'll only have a single `text-align: center` and no extra max-width, margin, or display styles.